contracts/TimeManagerV5.sol

```diff
@@ -14,11 +14,8 @@ contract TimeManagerV5 {
     /// @dev Sets true when contract is initialized
     bool private isInitialized;
 
-    /**
-     * @dev Retrieves the current slot
-     * @return Current slot
-     */
-    function() view returns (uint256) private _getCurrentSlot;
+    /// @dev Placeholder var that has to stay unused (following previous fix of _getCurrentSlot pointer)
+    uint256 private emptySlot;
```

Comment:
We are using `__deprecatedSlotX` in the RiskFundStore contract, for example: https://github.com/VenusProtocol/protocol-reserve/blob/develop/contracts/ProtocolReserve/RiskFundStorage.sol#L13

We could follow the same naming convention, if it works for you